### PR TITLE
Cms/383

### PR DESCRIPTION
**Pull request by Legai**

### CODE REVIEW

File: backend/app/views/pages/tit001Render.pug [hunk 1]
```diff
@@ -0,0 +1,10 @@
+doctype html
+html
+    head
+        meta(charset="utf-8")
+        link(rel="stylesheet" href="css/tit001Style.css")
+    body
+        div(class=divParams[0])
+            div(class=divParams[1]) !{text[0]}
```
Comment (IgorZiegler) [inline backend/app/views/pages/tit001Render.pug]: Use iteration

File: backend/app/views/pages/tit001Render.pug [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+            div(class=divParams[2]) !{text[1]}
+            div(class=divParams[3]) !{text[2]}
```
Comment (mvahrameev) [inline backend/app/views/pages/tit001Render.pug]: Use pug iteration methods

File: backend/public/css/tit001Style.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+main {
+  position: relative;
+}
+
+.container {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+
```
Comment (IgorZiegler) [inline backend/public/css/tit001Style.css]: Remove empty lines

File: backend/public/css/tit001Style.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.text {
+    font-weight: bold;
+    font-size: 2.8125em;
+
+    color: white;
+    text-align: center;
+}
+.Content {
```
Comment (mvahrameev) [inline backend/public/css/tit001Style.css]: To lowercase

File: backend/public/css/tit001Style.css [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    position: relative;
+    margin-top: 30px;
+
+    font-weight: lighter;
+    text-align: center;
+    font-size: 1.25em;
```
Comment (IgorZiegler) [inline backend/public/css/tit001Style.css]: Why do you use `em` here?

File: backend/routes/router.js [hunk 6]
```diff
@@ -1,7 +1,7 @@
 const express = require('express');
-const index = require('../app/controllers/index');
-const router = express.Router();
+const tit001Controller = require('../app/controllers/tit001Controller');
+const tit001Controller = express.Router();
```
Comment (mvahrameev) [inline backend/routes/router.js]: Hmm, maybe `const router = express.Router();` should be here?

File: backend/app/views/pages/tit001Render.pug [hunk 7]
```diff
@@ -0,0 +1,10 @@
+doctype html
+html
+    head
+        meta(charset="utf-8")
+        link(rel="stylesheet" href="css/tit001Style.css")
+    body
+        div(class=divParams[0])
```
Comment (mvahrameev) [inline backend/app/views/pages/tit001Render.pug]: Why do you use template variables for class names? You can jsut write:
```jade
.container
    .text !{text[0]}
    .line !{text[1]}
    .content !{text[2]}
```

**Pull request closed without merging**